### PR TITLE
Add a test to check if pfcwd gets started after load_minigraph

### LINKED ISSUE #4740: Add a test to check if pfcwd gets started after load_minigraph
**Description**

Details in Azure/sonic-utilities#1937

**Steps to reproduce the issue:**
1. Issue a "config load_minigraph" and check if pfcwd is running using the "pfcwd show config" command. Pfcwd should be started on all active ports

### PR DESCRIPTION
**Pull request by ZhaohuiS**
Signed-off-by: Zhaohui Sun <zhaohuisun@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
To fix #4740.
Add a test to check if pfcwd gets started after load_minigraph.

#### How did you do it?
Add a new test case test_default_cfg_after_load_mg, steps are:
1. Add a fixture to enable pfcwd configuration in init_cfg.json file.
2. run shell command: config load_minigraph -y
3. Check if pfcwd gets started with command: pfcwd show config

#### How did you verify/test it?
Run test case test_default_cfg_after_load_mg in tests/pfcwd/test_pfc_config.py.

### CODE REVIEW

File: tests/pfcwd/test_pfc_config.py [hunk 1]
```diff
@@ -268,3 +329,27 @@ def test_high_restore_time_cfg(self, duthosts, rand_one_dut_hostname):
...
+            duthost(AnsibleHost): instance
+
+        Returns:
+            None
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+
+        duthost.shell('config load_minigraph -y &>/dev/null', executable="/bin/bash")
+        # sleep 20 seconds to make sure configuration is loaded
+        time.sleep(20)
```
Comment (neethajohn) [inline tests/pfcwd/test_pfc_config.py:349]: load minigraph using this func "config_reload(duthost, config_source='minigraph')"

Comment (ZhaohuiS) [reply tests/pfcwd/test_pfc_config.py]: Done.

File: tests/pfcwd/test_pfc_config.py [hunk 2]
```diff
@@ -112,6 +114,65 @@ def cfg_setup(setup_pfc_test, duthosts, rand_one_dut_hostname):
...
+
+    Returns:
+        original value of default_pfcwd_status
+    """
+    output = duthost.shell("cat /etc/sonic/init_cfg.json | grep default_pfcwd_status")['stdout']
+    matched = re.search('"default_pfcwd_status": "(.*)"', output)
+    if matched:
+        original_value = matched.group(1)
+    else:
+        pytest.failed("There is no default_pfcwd_status in /etc/sonic/init_cfg.json.")
```
[the remark this answers is not in the document]
Comment (ZhaohuiS) [reply tests/pfcwd/test_pfc_config.py]: Done.

**Review by neethajohn — APPROVED** [review]

**Comment by antonptashnik** [comment]
@ZhaohuiS can you please say if default PFCWD config should be enabled on `config reload` ? I see the field `default_pfcwd_status` in config_db.json so I suspect it should be taken into account on config reload too to decide on PFCWD status. Thanks for the answer in advance!

**Comment by ZhaohuiS** [comment]
> @ZhaohuiS can you please say if default PFCWD config should be enabled on `config reload` ? I see the field `default_pfcwd_status` in config_db.json so I suspect it should be taken into account on config reload too to decide on PFCWD status. Thanks for the answer in advance!

@antonptashnik I think in my case, I covered `config load_minigraph` to address https://github.com/sonic-net/sonic-mgmt/issues/4740. It's probably better to check it for `config reload` as well, need the confirmation from @neethajohn

**Pull request merged**